include/constants/vars.h

```diff
@@ -94,7 +94,7 @@
 #define VAR_POKELOT_PRIZE_PLACE              0x404D
 #define VAR_UNUSED_0x404E                    0x404E // Unused Var
 #define VAR_LOTAD_SIZE_RECORD                0x404F
-#define VAR_0x4050                           0x4050
+#define VAR_LITTLEROOT_STATE                 0x4050
 #define VAR_ROUTE102_ACCESSIBLE              0x4051
```

Comment:
`VAR_OLDALE_TOWN_STATE`

Reply:
That's definitely Littleroot

Reply:
The comment's on the var currently named `VAR_ROUTE102_ACCESSIBLE`, which is linked to the dude checking his own footprints.